### ISSUE TITLE
Perform actions on jobs and pipelines

### ISSUE THREAD

**Issue by mathstuf** [nbedos/cistern#14]
It'd be nice to be able to restart a job or pipeline from the viewer.

Triggering a new one would also be nice, but is probably more involved (at least for GitLab).

(Thanks for the tool by the way; it's really awesome :) ).

**Comment by nbedos** [comment]
Yes, it would be nice to have both features. There are a couple of other changes I'd like to make first but this is definitely something I plan to work on.

> (Thanks for the tool by the way; it's really awesome :) ).

You're welcome!
